src/pages/explore/dfi/index.page.tsx

```diff
@@ -11,7 +11,7 @@ import { InitialTokenDistributionSection } from "./_components/InitialTokenDistr
 export default function ExploreDFI() {
   const { t } = useTranslation("page-explore-dfi");
   return (
-    <>
+    <div className="relative z-10">
```

Comment:
instead of having to add `z-10` for all other components, why not we place the generic bg as `z-index: -1`, then custom bg as `z-index: -2`, so that all other components will have `z-index: 0` by default

Reply:
Refactored!